controllers/gatling_controller.go

```diff
@@ -588,6 +598,11 @@ func (r *GatlingReconciler) newGatlingReporterJobForCR(gatling *gatlingv1alpha1.
 		},
 		Spec: batchv1.JobSpec{
 			Template: corev1.PodTemplateSpec{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:        r.getObjectMeta(gatling).Name,
+					Labels:      utils.AddMapValue("type", "reporter", r.getObjectMeta(gatling).Labels, false),
```

Comment:
@S-mishina only one more suggestion. 
I think in this case either value (true or value) is OK but why you put `false` to reporter case while you put `true` to runner case?  I think it's better to have consistency, by which I mean you should put the same value (true or false) to both runner and reporter

Reply:
@yokawasa 
Thanks for the review.
Sorry, I forgot to test and add back the values.
Correction.
Please check.
https://github.com/st-tech/gatling-operator/pull/63/commits/35a33df51d841354089b90dafd62aa7dd549a8fe

Reply:
@S-mishina looks awesome!